superset-frontend/src/SqlLab/main.less

```diff
@@ -310,8 +310,8 @@ div.Workspace {
   padding: 10px;
 
   .schemaPane {
-    flex: 0 0 300px;
-    max-width: 300px;
+    flex: 0 0 400px;
```

Comment:
This might be good to put on a "convert to Emotion" list somewhere, to have less LESS, but not worried about it for now!